recipes/sfml/3.0.0/test_package/CMakeLists.txt

```
@@ -0,0 +1,31 @@
+cmake_minimum_required(VERSION 3.1)
+project(test_package LANGUAGES CXX)
+
+add_executable(${PROJECT_NAME} test_package.cpp)
+
+set(SFML_COMPONENTS system)
+set(SFML_TARGETS sfml-system)
+if(SFML_WITH_WINDOW)
+    target_compile_definitions(${PROJECT_NAME} PRIVATE SFML_WITH_WINDOW)
+    list(APPEND SFML_COMPONENTS window)
+    list(APPEND SFML_TARGETS sfml-window)
```

```suggestion
    list(APPEND SFML_COMPONENTS Window)
    list(APPEND SFML_TARGETS SFML::Window)
```
The component and target names changed in v3. This same pattern applies for all other component and target names. See the [SFML 3 migration guide](https://github.com/SFML/SFML/blob/master/migration.md#cmake-targets) for more information.